gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/validation/GroupValidationServiceImpl.java

```diff
@@ -70,14 +67,19 @@ public Set<String> validateAndSanitize(
             }
         }
 
+        /*
+         * check with the other development team if this is necessary, as this LOC
+         * iss responsible for resetting the group even after it has been removed for V4 APIs
+         */
```

Comment:
This piece of code is required at creation time.
But the "validateAndSanitize" method is used in both creation and update case.
2 possibilities:
 - extract this piece of code in a dedicated method and call it explicitly during creation
 - add a parameter in the "validateAndSanitize" method to execute the piece of code only during creation